make-release.sh

```diff
@@ -28,6 +28,32 @@ usage ()
   exit 1
 }
 
+checkForBlockerIssues()
+{
+    # check for blockers only if doing a 7.yy.0 release
+    if [[ ${CHE_VERSION} == *".0" ]]; then 
+        # install gh cli from https://github.com/cli/cli/releases/
+        sudo dpkg -i https://github.com/cli/cli/releases/download/v2.2.0/gh_2.2.0_linux_amd64.deb
+
+        # find blockers for a given milestone
+        # BLOCKERS_THIS_MILESTONE="$(gh issue list -R eclipse/che -l "severity/blocker" -s "open" -m "${CHE_VERSION%.*}" --json "createdAt,updatedAt,author,title,url,milestone" | jq -r '.[]')"
+
+        # find any blockers, including those unassigned to milestones
+        BLOCKERS_ANY="$(gh issue list -R eclipse/che -l "severity/blocker" -s "open" --json "createdAt,updatedAt,author,title,url,milestone" | jq -r '.[]')"
```

Comment:
maybe it's simpler to call the API directly ?
```
$ curl -s "https://api.github.com/repos/eclipse/che/issues?labels=severity/blocker" | jq -r '.'
```

Reply:
simpler perhaps. But how can I pass in status (open/closed) or milestone to that query?

Reply:
By default the api is returning only open issues

But you can add ?state=closed for example to get closed issues or state=open

The syntax is longer than the direct API call :-)

Reply:
And for your request you're only interested in the array length so I don't know why you might want to select fields anyway

Reply:
The idea was to provide useful information to the person who has to then chase down the blocker issues.

```
milestone="7.39"
milestoneID=$(curl -s "https://api.github.com/repos/eclipse/che/milestones?sort_on=due_on&direction=desc&state=open" | jq -r --arg milestone $milestone '.[]|select(.title==$milestone)|.number' 2>&1) # 151

curl -s "https://api.github.com/repos/eclipse/che/issues?labels=severity/blocker&state=closed&milestone=${milestoneID}" | jq -r '.[]|[.created_at,.updated_at,.milestone.title,.url,.user.login,.title] | @tsv'

2021-11-02T13:53:01Z	2021-11-03T11:50:14Z	7.39	https://api.github.com/repos/eclipse/che/issues/20716	vinokurig	Che-theia editor docker image build fails
2021-10-18T08:19:51Z	2021-11-03T11:44:38Z	7.39	https://api.github.com/repos/eclipse/che/issues/20644	sleshchenko	Che should supply devfile v2 in Che Server mode (by converting it to devfile v1)
```